lib/mtg_cli/card_display.rb

```diff
@@ -0,0 +1,82 @@
+module MtgCli
+  class CardDisplay
+    LINE_COLUMNS = 40
+
+    attr_reader :card
+
+    def initialize(card)
+      @card = card
+      @traits = card.traits.map(&:to_sym)
+    end
+
+    def to_s
+<<-CARD
+
+#{header(card.name, card.mana_cost)}
+
+#{card.type}
+
+#{wrap(card.text)}
+
+#{pt(card.power, card.toughness)}
+CARD
+    end
+
+    def verbose
+<<-CARD
+
+#{header(card.name, card.mana_cost)}
+
+#{card.type}
+
+#{wrap(card.text)}
+
+#{TextEffects.italic(wrap(card.flavor))}
+
+#{pt(card.power, card.toughness)}
+
+Artist: #{card.artist}
+Rarity: #{card.rarity}
+Sets: #{card.set_name.join(', ')}
+CARD
+    end
+
+    private
+
+    def wrap(text)
+      text.gsub(/(.{1,40})(\s+|\z)/, "\\1\n").chomp("\n")
+    end
+
+    def header(name, cost)
+      cost = cost.tr('{}', '')
+      spaces = ' ' * (LINE_COLUMNS - (name.size + cost.size))
+      TextEffects.underline("#{name}#{spaces}#{cost}")
+    end
+
+    #This works, but I think it looks bad. We can rethink
+    def colorize_mana_cost(cost)
+      colored_cost = ""
+      cost.each_char do |c|
+        if c == "U"
```

Comment:
@Magisus Yeah it should be removed. If you want to save what you have at the moment you could open up a new branch, something like `text_effects`?